app/services/hyrax/quick_classification_query_decorator.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# OVERRIDE Hryax v5.0.0rc2
+
+module Hyrax
+  module QuickClassificationQueryDecorator
+    # OVERRIDE: only use work types that are enabled in the current tenant
+    # @param [::User] user the current user
+    # @param [#call] concern_name_normalizer (String#constantize) a proc that translates names to classes
+    # @param [Array<String>] models the options to display, defaults to everything.
+    def initialize(user,
```

Comment:
I think we could do:

```ruby
def initialize(user, models: Site.instance.available_works, **kwargs)
  super(user, **kwargs.merge(models: models))
end
```

That way we don't need to know about the other implementation details.